src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
Link to the stylesheet should have an extension of `.css` instead of `.scss`. `.scss` files are raw Sass files that need to be compiled into `.css` files before being linked in HTML.